tsdx.config.js

```diff
@@ -15,19 +15,26 @@ module.exports = {
       case 'umd':
         delete config.external
         config.output.indent = false
-        config.plugins.push(
+        config.plugins.unshift(
           replace({
-            '// UMD-DEV-ONLY: ': ''
+            '// UMD-ONLY: ': '',
+            delimiters: ['', '']
```

Comment:
This replacement was never matching due to it being constrained to word boundaries as [explained here](https://github.com/rollup/plugins/tree/master/packages/replace#word-boundaries).